packages/utils/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@web3modal/utils",
+  "version": "3.1.0",
+  "type": "module",
+  "main": "./dist/esm/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist",
+    "!tsconfig.tsbuildinfo"
+  ],
+  "scripts": {
+    "build:clean": "rm -rf dist",
+    "build": "npm run build:clean; tsc --build",
+    "watch": "npm run build:clean; tsc --watch",
+    "typecheck": "tsc --noEmit",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
+  },
+  "dependencies": {
+    "@web3modal/polyfills": "3.1.0",
+    "@web3modal/scaffold": "3.1.0"
+  },
+  "peerDependencies": {
```

Comment:
This is not gonna work, utils should have no reliance on either wagmi or ethers, so if something in utils is using imports from these, those need to be moved to their respective packages / duplicated with diff types etc if needed.